torch/ao/quantization/dynamic_tracing/auto_trace_rewrite.py

```diff
@@ -0,0 +1,230 @@
+import copy
```

Comment:
the file name makes it sound like its version 2.0 of auto_trace.py, maybe should be auto_trace_rewriter.py?

Reply:
sure